nsxt/resource_nsxt_policy_service_test.go

```diff
@@ -921,3 +1045,72 @@ resource "nsxt_policy_service" "test" {
   }
 }`, serviceName, serviceName, alg, sourcePorts, destPort)
 }
+
+func testAccNsxtPolicyNestedServiceCreateTemplate(serviceName string, nestedServiceEntryName string) string {
+	return fmt.Sprintf(`
+resource "nsxt_policy_service" "test" {
+  description  = "Nested service"
+  display_name = "%s"
+
+  nested_service_entry {
+    display_name        = "%s"
+    description         = "Entry-1"
+	nested_service_path = "/infra/services/%s"
```

Comment:
This is also problematic for global manager, since the path would be `/global-infra/services/HTTP`. You can use `testAccAdjustPolicyInfraConfig` for converting infra `path` to `global-infra`. Thank you!